src/models/management-account.js

```diff
@@ -37,6 +37,15 @@ export default class ManagementAccount extends ManagementModel {
       })
       .catch(err => Promise.reject(err));
   }
+  tokenInfo(access_token) {
+    return this.connection
+      .request({
+        method: 'POST',
+        path: `/a/${this.appId}/${this.constructor.collectionName}/${this.id}/token-info`,
+        body: { access_token: access_token ? access_token : this.connection.accessToken },
```

Comment:
I realized this was only working in the test b/c the `access_token` is assigned as the `client_secret` when creating a Nylas connection and I used the `client_secret` value as the `access_token` in the test. Since this is on the Account Management model, there's no way I can auto-populate the access token. The access token is only available when you make an account instance with: `nylas = Nylas.with(ACCESS_TOKEN)` so not going to auto-populate.